theme/static/css/hydroshare_core.css

```diff
@@ -983,33 +990,38 @@ li.account p{
 	table-layout: fixed;
 }
 
-.authors-wrapper > a:not(:first-child)::before, .authors-wrapper span:not(:first-child):not(.ui-sortable-helper) a::before {
+.authors-wrapper > a:not(:first-child)::before,
```

Comment:
is the use of `child` selectors robust enough? Absent specific ids, always good to make sure it's not fragile and will withstand future refactoring.

Reply:
Yes, this works perfectly for what it's trying to do. This adds a separator before each element and thus the first one doesn't need it because there is none before.